src/index.ts

```diff
@@ -0,0 +1,5 @@
+import * as utils from './utils'
+import * as constants from './constants'
+
+// eslint-disable-next-line import/prefer-default-export
+export { utils, constants }
```

Comment:
Wouldn't `export default { utils, constants }` do the trick as well? Then you wouldn't need to add the linter comment.

Reply:
Meh, matter of personal preference though... I dont like to import stuff like `import bla from 'amsterdam-react-maps'; const {constants} = bla ....` Unless we have a main component that we want to export already, that can be our default but for now I cannot think about one